extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -70,18 +96,27 @@ oppia.directive('oppiaInteractiveImageClickInput', [
             }
           };
           $scope.onMousemoveImage = function(event) {
-            var image = $($element).find('.oppia-image-click-img');
-            $scope.mouseX = (event.pageX - image.offset().left) / image.width();
-            $scope.mouseY = (event.pageY - image.offset().top) / image.height();
-            $scope.currentlyHoveredRegions = [];
-            for (var i = 0; i < imageAndRegions.labeledRegions.length; i++) {
-              var labeledRegion = imageAndRegions.labeledRegions[i];
-              var regionArea = labeledRegion.region.area;
-              if (regionArea[0][0] <= $scope.mouseX &&
-                  $scope.mouseX <= regionArea[1][0] &&
-                  regionArea[0][1] <= $scope.mouseY &&
-                  $scope.mouseY <= regionArea[1][1]) {
-                $scope.currentlyHoveredRegions.push(labeledRegion.label);
+            $scope.$on('newCardAvailable', function(evt, data) {
+              if(data) {
+                $scope.deactivate = true;
+              }
+            });
+            if(!$scope.deactivate) {
+              var image = $($element).find('.oppia-image-click-img');
+              $scope.mouseX =
+                (event.pageX - image.offset().left) / image.width();
+              $scope.mouseY =
+                (event.pageY - image.offset().top) / image.height();
+              $scope.currentlyHoveredRegions = [];
+              for (var i = 0; i < imageAndRegions.labeledRegions.length; i++) {
```

Comment:
There's lots of code duplication here with the stuff above; factor it into a common method.

Reply:
Ok, ya, will do that

Reply:
Done.